lua/telescope/_extensions/dap.lua

```diff
@@ -234,9 +234,13 @@ local frames = function(opts)
 end
 
 return telescope.register_extension {
-  setup = function()
+  setup = function(ext_config)
     vim.cmd [[ highlight default link NvimDapSubtleFrame Comment ]]
 
+    if not vim.F.if_nil(ext_config.overwrite_pick_one, true) then
```

Comment:
I changed this to `true` by default so that the old behavior is maintained if no value for `overwrite_pick_one` is provided.